package.json

```diff
@@ -39,6 +39,11 @@
                     "type": "boolean",
                     "default": true,
                     "description": "Use spaces or tabs for indentation (False will ignore indentWidth)"
+                },
+                "angular-template-formatter.closeTagNewLine": {
+                    "type": "boolean",
+                    "default": false,
+                    "description": "Always place the closing element tag '>' on a new line."
```

Comment:
It may make more sense for this to be `angular-template-formatter.closeTagSameLine`

then it can say `"Never place the closing tag '>' on a new line."`

because technically with this setting we don't always place it on a new line, only if there are 2 or more attributes on the element.

Reply:
all righty, sounds good, changes incoming